gamemode/gametypes/hl2/mapscripts/d3_citadel_03.lua

```diff
@@ -31,24 +31,49 @@ function MAPSCRIPT:PostInit()
             end
         end
 
+        GAMEMODE:WaitForInput("logic_weapon_strip_strip", "Trigger", function(logic_strip)
+            timer.Simple(3.8, function()
```

Comment:
Ideally this should use util.RunDelayed which is cleared on round restarts unlike timers, this would otherwise run even afterwards. The function works like following:
```lua
    util.RunDelayed(function()
        print("Delayed action")
    end, CurTime() + 3.8)
```